hasjob/templates/sheet.html.jinja2

```diff
@@ -22,6 +22,5 @@
 {% endblock %}
 
 {% block layoutscripts %}
-  {%- if header_campaign %}{{ campaign_script() }}{% endif %}
-  {% block footerscripts %}{% endblock %}
+  {{ super() }}
 {% endblock %}
```

Comment:
We don't need to define a block if it _only_ contains `super()`.

Reply:
Removed.